HipChatAdmin/Public/Add-HipchatUserToRoom.ps1

```diff
@@ -48,7 +48,10 @@ function Add-HipchatUserToRoom{
 			# Check response status code #
 			if ($Call.StatusCode -eq '204') {
 				Write-Verbose "User $Name Added Successfully!"
-				Write-Output $Call.StatusCode
+                $OutputObject = New-Object -TypeName PSObject
+				$OutputObject | Add-Member -MemberType 'NoteProperty' -Name 'Name' -Value $name
+				$OutputObject | Add-Member -MemberType 'NoteProperty' `-Name 'StatusCode' -Value $Call.StatusCode
```

Comment:
Each one of these has a backtick between 'NoteProperty' and -Name. Is this intentional?